serial/serbridge.c

```diff
@@ -13,6 +40,15 @@
 
 #define SKIP_AT_RESET
 
+
+/*  ESP8266
+ */
+#define IROM			ICACHE_FLASH_ATTR
```

Comment:
Is there a reason for this? It seems just confusing. Imagine if every person defined their own macro for this and there were 10 different ones throughout the codebase...